### PR TITLE
Add stub cleanup helper

### LINKED ISSUE #38: Cleanup stubs
There is currently a problem with stubs being left in place when tests fail before reaching the unstub command. There is already an issue open for stubs to support cleanup jasonkarns/bats-mock#3 however it would be worth investigating if any cleanup could be done in a helper function.

### PR DESCRIPTION
**Pull request by jt-nti**
Add stub cleanup helper function for use in teardown functions to
remove any leftover stubs from failing testcases

Closes #38

Signed-off-by: James Taylor <jamest@uk.ibm.com>

### CODE REVIEW

File: test/test_helper.bash [hunk 1]
```diff
@@ -16,3 +16,12 @@ setup_script_dir() {
   mkdir -p "${SCRIPT_DIR}"
   cp -a "${src_dirname}/"* "${SCRIPT_DIR}"
 }
+
+cleanup_stubs() {
+  if stat ${BATS_TMPDIR}/*-stub-plan >/dev/null 2>&1; then
```
Comment (sanjayravindraibm) [inline test/test_helper.bash:21]: Do we only need to delete the *-stub-plan files, or also the *-stub-run files? Also, do we have to delete the files in the bin folder?

**Review by sanjayravindraibm — COMMENTED** [review]
Do we only need to delete the *-stub-plan files, or also the *-stub-run files? Also, do we have to delete the files in the bin folder?

File: test/test_helper.bash (hunk 1 above)
Comment (jt-nti) [reply test/test_helper.bash:21]: Deleting those files should be handled by calling unstub on line 24

Comment (sanjayravindraibm) [reply test/test_helper.bash:21]: I see. Looks good!

**Review by sanjayravindraibm — APPROVED** [review]
Good to go

**Pull request merged**